examples/autocomplete.amp.html

```diff
@@ -86,6 +87,132 @@ <h3>Dynamic data</h3>
         </div>
     </form>
 
+    <h3>Both data sources provided (should cause warning and display remote data)</h3>
+    <form method="post"
+        action-xhr="http://amp.localhost:8000/form/echo-json/post"
+        target="_blank">
+        <fieldset>
+            <label>
+                <span>Your hometown</span>
+                <amp-autocomplete filter="substring" min-characters="0"
+                    src="autocomplete-cities.example.json">
+                    <input type="text" name="city" required>
+                    <script type="application/json">
+                      { "items" : ["hello a", "hello b", "hello c"] }
+                    </script>
+                </amp-autocomplete>
+            </label>
+            <input name="submit-button" type="submit" value="Submit">
+        </fieldset>
+
+        <div submit-success>
+            <template type="amp-mustache">
+                Success! Mailing a postcard to {{city}}.
+            </template>
+        </div>
+    </form>
+
+    <h3>Neither data sources provided (should cause warning)</h3>
+    <form method="post"
+        action-xhr="http://amp.localhost:8000/form/echo-json/post"
```

Comment:
Can you help me understand why `amp.localhost` is used? Do you need to test cross origin functionality, or can it be directly on `localhost`?